### ISSUE TITLE
Consider webpack 5 support

### ISSUE THREAD

**Issue by plesiecki** [tivac/modular-css#848]
### Describe the problem

Seems like webpack plugin doesn't necessarily work with Webpack 5.

Currently plugin fails on `compiler.plugin` use.

```
TypeError: compiler.plugin is not a function
    at ModularCSS.apply (/repo/node_modules/@modular-css/webpack/plugin.js:37:14)
```

### Describe the proposed solution

Have you considered supporting Webpack 5?

### Alternatives considered

dunno

### Importance

I cannot use modular-css without it

**Comment by tivac** [comment]
I'll take a look at what is involved. I haven't used webpack in years so can't promise much.

**Comment by tivac** [comment]
Leaving open until it's published, but it's merged at least.

**Comment by tivac** [comment]
`webpack@5` support is released as `@modular-css/webpack@28.2.0`

https://github.com/tivac/modular-css/releases/tag/v28.2.0
